### PR TITLE
Add permissions policy

### PR DESCRIPTION
**Pull request by SaraVieira**
- Adds support for permissions policy meta tag https://www.w3.org/TR/permissions-policy-1/
- Drafts new release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.3.1 - 2023-01-24
```
Comment (ashleygwilliams) [inline CHANGELOG.md]: i think we should do 0.4.0 because this change is a new feature, not a bug fix

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+
+### 🎁 Features
+
+- **✨ Add support for Permissions-Policy meta tag- [SaraVieira], [PR23]**
+
+  This PR adds support for using the [`Permissions-Policy` meta tag](https://www.w3.org/TR/permissions-policy-1/) that is is used for defining a set of browser API's you do not wish your website to have.
```
Comment (ashleygwilliams) [inline CHANGELOG.md]:
```suggestion
  This PR adds support for using the [`Permissions-Policy` meta tag](https://www.w3.org/TR/permissions-policy-1/) that is used for defining a set of browser APIs you do not wish your website to have.
```

remove extra word, apostrophe

File: typed-html/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "axohtml"
-version = "0.3.0"
+version = "0.3.1"
```
Comment (ashleygwilliams) [inline typed-html/Cargo.toml]: same here, 0.4.0 not 0.3.1

**Review by ashleygwilliams — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (SaraVieira) [reply CHANGELOG.md]: fair, it was just so small it felt wrong but you are right

File: typed-html/Cargo.toml (hunk 3 above)
Comment (SaraVieira) [reply typed-html/Cargo.toml]: done

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**